201-vm-copy-index-loops/azuredeploy.json

```diff
@@ -61,7 +61,13 @@
       "sku": "2016-Datacenter",
       "version": "latest"
     },
-    "imageReference": "[variables(parameters('OS'))]"
+    "imageReference": "[variables(parameters('OS'))]",
+    "networkSecurityGroupName": "default-NSG",
+    "nsgOsPort": {
+      "Ubuntu": "22",
+      "Windows": "3389"
+    },
+    "nsgAllowPort": "[variables('nsgOsPort')[parameters('OS')]]"
```

Comment:
remove this and just use the expression inline

Reply:
Done.